### PR TITLE
Feature- Added icons to the top bar subMenu

### LINKED ISSUE #1500: Add icons to the top bar menu items
Currently there are no icons for the top bar menu sub-items:
image

It would be nice to add small icons before the items, e.g. open, save, export, undo, redo, zoom in, notices ...

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
Summary of changes
Added icons to the top bar menu items

closes #1500 
[ ] (https://github.com/opossum-tool/OpossumUI/pull/1669)
It would be nice to add small icons before the items, e.g. open, save, export, undo, redo, zoom in, notices ..

Context and reason for change
A feature requested

How can the changes be tested
Run it. yarn start.

@benedikt-richter @leslielazzarino here

Note: Please review the [guidelines for contributing](https://github.com/opossum-tool/OpossumUI/blob/main/CONTRIBUTING.md) to this repository.

### CODE REVIEW

File: src/ElectronBackend/main/getPath.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// SPDX-FileCopyrightText: Meta Platforms, Inc. and its affiliates
+// SPDX-FileCopyrightText: TNG Technology Consulting GmbH <https://www.tngtech.com>
```
Comment (benedikt-richter) [inline src/ElectronBackend/main/getPath.ts:2]: Here you can simply put:
```
// SPDX-FileCopyrightText: Tarun Samanta <your e-mail>

```

Comment (tarunsamanta2k20) [reply src/ElectronBackend/main/getPath.ts:2]: @benedikt-richter should i remove Meta Platfrom as well or only i have to add at line no. 2?

Comment (tarunsamanta2k20) [reply src/ElectronBackend/main/getPath.ts:2]: @benedikt-richter Let it be.

Comment (tarunsamanta2k20) [reply src/ElectronBackend/main/getPath.ts:2]: @benedikt-richter like this:
 // SPDX-FileCopyrightText: Meta Platforms, Inc. and its affiliates
// SPDX-FileCopyrightText: Tarun Samanta <tarunsamanta77@gmail.com>
//
// SPDX-License-Identifier: Apache-2.0

Comment (benedikt-richter) [reply src/ElectronBackend/main/getPath.ts:2]: @tarunsamanta2k20 the following would be appreciated:
```
// SPDX-FileCopyrightText: Tarun Samanta [tarunsamanta77@gmail.com](mailto:tarunsamanta77@gmail.com)
//
// SPDX-License-Identifier: Apache-2.0
```

**Review by benedikt-richter — APPROVED** [review]
Works nicely!
Thanks for adding the icons. It makes the menu much more enjoyable.

**Comment by benedikt-richter** [comment]
You can merge if you wish

**Pull request merged**